client/my-sites/customer-home/main.jsx

```diff
@@ -335,21 +346,26 @@ class Home extends Component {
 			siteSlug,
 			trackAction,
 			isStaticHomePage,
+			staticHomePageId,
 			showCustomizer,
 			hasCustomDomain,
 		} = this.props;
 
-		const siteTools = (
+		const editHomePageUrl =
+			isStaticHomePage && `/block-editor/page/${ siteSlug }/${ staticHomePageId }`;
+
+		const quickLinks = (
 			<div className="customer-home__boxes">
-				<ActionBox
-					onClick={ () => {
-						trackAction( 'my_site', 'add_page' );
-						page( `/page/${ siteSlug }` );
-					} }
-					label={ translate( 'Add a page' ) }
-					iconSrc={ pageIcon }
-				/>
 				{ isStaticHomePage ? (
+					<ActionBox
+						onClick={ () => {
+							trackAction( 'my_site', 'edit_homepage' );
+							page( editHomePageUrl );
+						} }
+						label={ translate( 'Edit homepage' ) }
+						iconSrc={ imagesIcon }
```

Comment:
I'm really unsure what to use here. It looks odd to use the same icon twice, so I've used the images icon for now, but I'm not sure it really fits?

<img width="340" alt="Screenshot 2020-02-08 at 13 36 09" src="https://user-images.githubusercontent.com/43215253/74086136-f9c19b00-4a77-11ea-9a06-f064e670923e.png">

Reply:
Hm, I see your point. I'm a little torn on these icons; I'm not sure how much we want them to convey actual information, or are they just basically filling in as visual appeal. I personally don't think another "page" icon is a problem here, but @danhauk will have a definitive answer.

Reply:
According to https://github.com/Automattic/wp-calypso/pull/39133#issuecomment-579838474, we'd like to show a "desktop computer" icon, but I don't think we currently have that icon in our set of icons.

Reply:
That would work nicely! @danhauk Is it possible to share that icon design or add it to https://github.com/Automattic/wp-calypso/tree/master/client/assets/images/customer-home?